src/arbitrary/constantFrom.ts

```diff
@@ -2,6 +2,21 @@ import { Arbitrary } from '../check/arbitrary/definition/Arbitrary';
 import { convertFromNext } from '../check/arbitrary/definition/Converters';
 import { ConstantArbitrary } from './_internals/ConstantArbitrary';
 
+type Arrayfy<T> = T extends any[] ? T : T[];
+type Elem<T> = T extends any[] ? T[number] : T;
+
+/**
+ * For one `...values` values - all equiprobable
+ *
+ * **WARNING**: It expects at least one value, otherwise it should throw
+ *
+ * @param values - Constant values to be produced (all values shrink to the first one)
+ *
+ * @remarks Since 0.0.12
+ * @public
+ */
+function constantFrom<T>(...values: T[]): Arbitrary<T>;
```

Comment:
I think you wanted to add `=never` to cover the last (not so) _problematic_ typing.

```suggestion
function constantFrom<T = never>(...values: T[]): Arbitrary<T>;
```